linux_os/guide/services/ssh/ssh_server/sshd_disable_pubkey_auth/rule.yml

```diff
@@ -0,0 +1,30 @@
+documentation_complete: false
```

Comment:
The build failures is because the rule is marked as `documentation_complete: false`.
Marking as `true` will resolve build issues.

Now I wonder if totally failing the build is what should happen.
At least it made evident that this rule would not be part of the DataStream.

Reply:
I'll mark `documentation_complete` true for now, but I didn't want to imply the doc was done when I'm not sure it actually is.....